modules/SofaConstraint/BilateralInteractionConstraint.cpp

```diff
@@ -202,7 +202,7 @@ class BilateralInteractionConstraintSpecialization<RigidImpl>
         const helper::vector<int> &m2Indices = self.m2.getValue();
 
         unsigned min = std::min(m1Indices.size(), m2Indices.size());
-        const  typename BilateralInteractionConstraint<T>::VecDeriv& restVector = self.restVector.getValue();
+        // const  typename BilateralInteractionConstraint<T>::VecDeriv& restVector = self.restVector.getValue();
```

Comment:
I'm not a big fan of commented code, it's hard to maintain. 
Shouldn't this be simply deleted?